sphinx/ext/inheritance_diagram.py

```diff
@@ -137,14 +137,15 @@ class InheritanceGraph:
     graphviz dot graph from them.
     """
     def __init__(self, class_names: List[str], currmodule: str, show_builtins: bool = False,
-                 private_bases: bool = False, parts: int = 0, aliases: Dict[str, str] = None,
-                 top_classes: List[Any] = []) -> None:
+                 private_bases: bool = False, parts: int = 0, style: str = "default",
```

Comment:
Please append a new parameter to the tail of the argument list to keep compatibility.